connections/src/app/core/services/connection.service.ts

```diff
@@ -1,9 +1,17 @@
 import { Injectable } from '@angular/core';
+import { environment } from 'src/environments/environment';
 
 @Injectable({
   providedIn: 'root'
 })
 export class ConnectionService {
 
+  envName!: string;
+  
   constructor() { }
+  
```

Comment:
@Siddharth-Tiwari1712 I see it added here. But where is it used?
I saw removed from app. component html template. 
And it should so up somewhere?
Maybe merely console.log so we can go check it there